### ISSUE TITLE
Suggestion for Mass Onboarding 

### ISSUE THREAD

**Issue by IgbozeIsrael** [alpha-fi/cheddar-ecosystem#6]
With nada bot verification, it is an issue to onboard lots of people. I have a suggestion that will lead to refactoring a few things. 
1. Make a referral link that has a wallet connect link attached, This can be done via Sharddog or Meteor wallet 
2.  Make an off-chain Cheese token or points where users accumulate by playing games. When it is accumulated up to a certain number, the backend sends a message to the smart contract and they can be able to claim $cheddar on-chain. At this point, the contract can only allow nada bot verified wallets to claim rewards. 
In summary, nada bot verification should be removed from the initial criteria to play games off-chain until they have to claim rewards on-chain
